src/defaultsettings.cpp

```diff
@@ -76,11 +76,19 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("tooltip_show_delay", "400");
 
 	// Some (temporary) keys for debugging
+	#ifdef NDEBUG
```

Comment:
Also concerning code style, preprocessor statements are usually without indentation regardless of all other stuff.